packages/ariakit/src/heading/__examples__/heading/style.css

```
@@ -0,0 +1,15 @@
+.headingContainer{
+    @apply flex flex-col
+}
+
+h1{
+    @apply text-xl
+}
+
+h2{
+    @apply text-lg mt-4
+}
+
+h3{
+    @apply text-base mt-4
+}
```

Updating the wrapper class and some formatting. It would be awesome if prettier worked on CSS files.
```suggestion
.wrapper {
  @apply flex flex-col
}

h1 {
  @apply text-xl
}

h2 {
  @apply text-lg mt-4
}

h3 {
  @apply text-base mt-4
}

```